<launch>
    <node pkg="pointcloud_utils" type="img_saver.py" name="image_saver"
          required="true" output="screen" cwd="node">
      	   <param name="img_topic" 		value="/grid_image/innoviz" />
      	   <param name="folder_name" 		value="innoviz_images" />
    </node>

    <node pkg="pointcloud_utils" type="img_saver.py" name="image_saver2"
          required="true" output="screen" cwd="node">
      	   <param name="img_topic" 		value="/grid_image/velodyne" />
      	   <param name="folder_name" 		value="velodyne_images" />
    </node>
</launch>